lib/conjur/debify.rb

```diff
@@ -315,11 +315,13 @@ def build_test_image(appliance_image_id, project_name)
 
 COPY #{deb} /tmp/
 
+RUN dpkg --install /tmp/#{deb}
+RUN dpkg --force all --purge conjur-#{project_name} || true
```

Comment:
Shouldn't we `rm` the source-installed package, as below, before installing the deb for the first time?